manifests/repo/yum.pp

```diff
@@ -8,12 +8,16 @@
     if $sensuclassic::repo_source {
       $url = $sensuclassic::repo_source
     } else {
+      $major = $facts.dig('os', 'release', 'major')
+      $family = $facts.dig('os', 'family')
       if $::operatingsystem == 'Amazon' {
-        if $facts['os']['release']['major'] =~ /^201\d$/ {
+        if $major =~ /^201\d$/ {
           $releasever = '6'
         } else {
           $releasever = '7'
         }
+      } elsif $family == 'RedHat' and $major == '8' {
+        $releasever = '7'
```

Comment:
comment why it is using the EL7 packages

Reply:
Comment added